types/report-renderer.d.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+declare global {
+  module LH.Renderer {
+    export function renderFullReport(lhr: LH.Result, options?: ReportRendererOptions): HTMLElement;
+    export function renderReportComponents(
+      lhr: LH.Result,
+      options?: ReportRendererOptions
+    ): ReportComponents;
+
+    export function getFinalScreenshot(): void | string;
```

Comment:
this was always ambiguous to me whether `data: image/jpeg;base64,f64acde...` is returned or just `f64acde...` is returned.

While we're improving things... WDYT about `getFinalScreenshotDataURL`? or an object return `{mimeType: 'image/jpeg', base64: 'f64acde...'}`?

Reply:
`getFinalScreenshotDataURL` or `getFinalScreenshotURL` for me dawg. don't make people construct a url :)

Reply:
great call. done.